src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java

```diff
@@ -131,16 +124,15 @@ public void testProperFileExtension() throws Exception {
      */
     @Test
     public void testNoAuditEventsWithoutFilters() throws Exception {
-        final DefaultConfiguration checkConfig = createModuleConfig(OneTopLevelClassCheck.class);
         final String[] expected = {
-            "5:1: " + getCheckMessage(OneTopLevelClassCheck.class,
-                    OneTopLevelClassCheck.MSG_KEY, "InputTreeWalkerInner"),
+            "10:1: " + getCheckMessage(OneTopLevelClassCheck.class,
+                        OneTopLevelClassCheck.MSG_KEY, "InputTreeWalkerInner"),
         };
         try (MockedConstruction<TreeWalkerAuditEvent> mocked =
-                 Mockito.mockConstruction(TreeWalkerAuditEvent.class, (mock, context) -> {
-                     throw new CheckstyleException("No audit events expected");
-                 })) {
-            verify(checkConfig, getPath("InputTreeWalker.java"), expected);
+                     Mockito.mockConstruction(TreeWalkerAuditEvent.class, (mock, context) -> {
```

Comment:
Restore indentation as it was before

Reply:
@romani done. I should disable IntelliJ reformate on paste